DOCKER.md

```diff
@@ -2,12 +2,14 @@
 You can package Jenkinsfile Runner with a specific Jenkins image and turn that into a Docker image.
 This way, you can ensure people are running Jenkinsfile in the specific Jenkins environment, for example one that's identical to your production environment.
 
-## Building
+## Building the Vanilla image
+This repository offers a Dockerfile to generate your docker image.
+
 Edit the `plugins.txt` to include any plugins you'd like to install. See [the Jenkins Docker README](https://github.com/jenkinsci/docker#preinstalling-plugins) for more information.
 
 Then, build the Jenkinsfile Runner image like this:
 
-```
+```bash
```

Comment:
Is this for Github's formatting?

Reply:
That enables syntax highlighting. I've seen it was used in other parts of the documentation and I've unified its use